torch/csrc/autograd/engine.cpp

```diff
@@ -1545,6 +1578,24 @@ void GraphTask::stash_current_streams() {
   }
 }
 
+// Remembers current streams on all devices where a context has been created for
+// privateuse1
+void GraphTask::stash_current_privateuse1_streams() {
```

Comment:
nit: can we rename the existing `stash_current_streams` to `stash_current_cuda_streams` 
(don't forget to update the comment above it as well)

Reply:
Ok，I changed the name and added privateuse1 to the comments.